client/my-sites/checkout/composite-checkout/hooks/use-create-payment-complete-callback.tsx

```diff
@@ -101,6 +103,7 @@ export default function useCreatePaymentCompleteCallback( {
 
 			// In the case of a Jetpack product site-less purchase, we need to include the blog ID of the
 			// created site in the Thank You page URL.
+			// TODO: It does not seem like this would be needed for Akismet, but marking to follow up
```

Comment:
Do you know why this is needed for jetpack siteless checkout? The page seems to work fine if I remove that param (for jetpack siteless)

Anyway, from what I understand so far, I also do not think that this is necessary for Akismet

Reply:
At a high level, it has to do with assigning the purchased Jetpack license from the thank you page - I think when you are actually assigning the license to one of your Jetpack sites, the temporary site ID from the purchase is also needed for some of the steps in that process.

Since the Akismet purchases won't have license management/ assignment steps, we shouldn't need the temporary site ID.